src/new-apps/components/AppInProgressDeleteDialog/AppInProgressDeleteDialog.test.tsx

```diff
@@ -0,0 +1,72 @@
+import Wrapper from "@test/wrapper";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+
+import AppInProgressDeleteDialog from "./AppInProgressDeleteDialog";
+import msgs from "./messages";
+
+describe("Apps AppInProgressDeleteDialog", () => {
+  it("displays action text with app name when app name passed", () => {
+    // Arrange
+    const name = "Test App";
+    render(
+      <Wrapper>
+        <AppInProgressDeleteDialog
+          confirmButtonState="default"
+          open={true}
+          name={name}
+          onClose={jest.fn()}
+          onConfirm={jest.fn()}
+        />
+      </Wrapper>,
```

Comment:
**issue**: please do not use a wrapper like this within the unit test, you need to mock certain things that you really rely on in your code and deliver only that as your mock. Also thing about testing what's inside that modal, instead of modal itself

Using that sort of wrapper is not really a unit test, it feels like a functional one, as you demand more application context to be delivered. Since it can come from many providers, states and so on, it's hard to debug that test later